frontend_tests/node_tests/search_suggestion.js

```diff
@@ -478,15 +478,15 @@ run_test('empty_query_suggestions', () => {
     function describe(q) {
         return suggestions.lookup_table.get(q).description;
     }
-    assert.equal(describe('is:private'), 'Private messages');
-    assert.equal(describe('is:starred'), 'Starred messages');
-    assert.equal(describe('is:mentioned'), '@-mentions');
-    assert.equal(describe('is:alerted'), 'Alerted messages');
-    assert.equal(describe('is:unread'), 'Unread messages');
-    assert.equal(describe('sender:myself@zulip.com'), 'Sent by me');
-    assert.equal(describe('has:link'), 'Messages with one or more link');
-    assert.equal(describe('has:image'), 'Messages with one or more image');
-    assert.equal(describe('has:attachment'), 'Messages with one or more attachment');
+    assert.equal(describe('is:private'), 'Translated: private messages');
```

Comment:
I don't really know how this works, but if adding the `Translated:` tag it will search in English and other available translations, then adding this on a preparatory commit wouldn't break the code and make the commit structure better.